### PR TITLE
feat: added full list of environement credentials

### PR DESCRIPTION
**Pull request by vickywane**
This pull request modifies the `README.md` file to contain a full list of environment variables used within this API.

### CODE REVIEW

**Comment by vickywane** [comment]
@ivelin The CI keeps breaking due to the Netlify deployment bug. 

I am still in touch with the Netlify support engineers and it's something they are actively working on. Till they provide a fix we have to ignore it.

**Comment by ivelin** [comment]
@vickywane 

Please clarify in the document which ones are required by github during CI build time and which ones are required by Netlify during function runtime?

When the code is build and run locally, all variables are in the same local environment. But that is not the case when the build is done via CI and deployment is separately done on Netlify where the functions run.

**Comment by vickywane** [comment]
@ivelin I went through the readme explanation. 

I can see the explanations on environment credentials are quite confusing so I rephrased them and clarified the needed environment variables. 

The environment variables you need for deployment and also running locally are in the `Running Locally` section. For the Github Actions, the env vars are list in the `CI / CD Pipeline` section of the README.

File: README.md [hunk 1]
```diff
@@ -9,26 +9,48 @@ The image below shows the architecture of these serverless functions in connecti
...
-
-Continuous integration for this project is managed using [GitHub Actions](https://github.com/features/actions), driven using the steps in the `.github/workflows/ci.yaml` file. The following sensitive credentials used within the CI jobs are managed using [Github Secrets](https://docs.github.com/en/actions/reference/encrypted-secrets);
+Continuous integration for this project is managed using [GitHub Actions](https://github.com/features/actions), driven using the steps in the `.github/workflows/ci.yaml` file.
+The following sensitive credentials are used within the CI jobs are stored using [Github Secrets](https://docs.github.com/en/actions/reference/encrypted-secrets) and are retrieved and used within each workflow build;
 
-- STRIPE_KEY
 - GITHUB_TOKEN
```
Comment (ivelin) [inline README.md:56]: @vickywane 

The readme is still confusing.

I see a `## Running Locally` section and a `## CI / CD Pipeline` section, but where is the live deployment section? It needs to show the runtime deployment environment variables that Netlify needs to provide to the cloud functions. For example Stripe key, Auth0 keys and Mailgun keys.

**Review by ivelin — CHANGES_REQUESTED** [review]
@vickywane please read the README again from the point of view of a new user or contributor. What are the essential pieces missing there to run and deploy successfully?

File: README.md (hunk 1 above)
Comment (vickywane) [reply README.md:56]: Basically, the credentials listed in the `Running Locally` section are used when deployed to Netlify. The only difference is the variables are stored in a `.env` file locally and are stored in Netlify when deployed.

Would it be sufficient to rename that section to `Executing Serverless Functions` and also include a text indicating that the variables are also used within Netlify?

**Comment by ivelin** [comment]
I would suggest instead adding a separate section after CI that focuses on
the cloud deployment environments such as Netlify.

You may even want to clarify in each section that developers should use their test keys for local and preview deployments. And respectively deployment keys WITH A LOT OF CAUTION when deploying live.

**Review by ivelin — APPROVED** [review]
@vickywane looks good. I'll merge this because its a README edit. Still need to figure out what's wrong with netlify pre-deployment before the next code PR.

**Comment by ivelin** [comment]
@vickywane docs changes should be prefixed in the commit and PR subject with "docs: ..." not "feat: ...".

**Comment by vickywane** [comment]
Alright. 

I would note these against any new next PR. :)

**Pull request merged**